docs/operator/configuring-serving-cr.md

```diff
@@ -0,0 +1,371 @@
+---
+title: "Configuring Serving Operator Custom Resource"
+weight: 10
+type: "docs"
+aliases:
+- /docs/operator/configuring-serving-cr/
+---
+
+# Configuring Serving Operator Custom Resource
+
+You are able to configure Knative Serving with the following options:
+
+* [All the ConfigMaps](#all-the-configmaps)
+* [Private repository and private secret](#private-repository-and-private-secrets)
+* [SSL certificate for controller](#ssl-certificate-for-controller)
+* [Knative ingress gateway](#configuration-of-knative-ingress-gateway)
+* [Cluster local gateway](#configuration-of-cluster-local-gateway)
+
+Currently, Knative operators are __NOT__ able to configure the following options for Knative:
```

Comment:
Make this a note instead of adding __NOT__  formatting.
e.g.

**NOTE:** High availability and Kubernetes level policies cannot be configured using the Knative operators."